src/patternfly/components/ChipGroup/examples/ChipGroup.md

```diff
@@ -5,269 +5,264 @@ cssPrefix: pf-c-chip-group
+  {{/chip}}
+  {{#> chip}}
+    {{#> chip-text chip-text--attribute=(concat 'id="' chip-group--id 'chip_two_toolbar_collapsed"')}}
+      Chip two
+    {{/chip-text}}
+    {{#> button button--modifier="pf-m-plain" button--attribute=(concat 'aria-labelledby="' chip-group--id 'remove_chip_two_toolbar_collapsed ' chip-group--id 'chip_two_toolbar_collapsed" aria-label="Remove" id="' chip-group--id 'remove_chip_two_toolbar_collapsed"')}}
+      <i class="fas fa-times" aria-hidden="true"></i>
+    {{/button}}
+  {{/chip}}
+  {{#> chip}}
+    {{#> chip-text chip-text--attribute=(concat 'id="' chip-group--id 'chip_three_toolbar_collapsed"')}}
+      Chip three
+    {{/chip-text}}
+    {{#> button button--modifier="pf-m-plain" button--attribute=(concat 'aria-labelledby="' chip-group--id 'remove_chip_three_toolbar_collapsed ' chip-group--id 'chip_three_toolbar_collapsed" aria-label="Remove" id="' chip-group--id 'remove_chip_three_toolbar_collapsed"')}}
+      <i class="fas fa-times" aria-hidden="true"></i>
+    {{/button}}
+  {{/chip}}
 {{/chip-group}}
 ```
 
-### Overview
-A chip-group used in a toolbar require the modifier `.pf-m-toolbar` which styles the group's background-color and border-radius. Multiple chip-groups can exist in the toolbar depending on the size of the group that is being filtered from and its parent container should handle the spacing between the chip groups. Categories can be labelled by using  `.pf-c-chip-group__label` and its heading level should be based on the context in which this component exists on the page. When groups of chips overflow they will wrap to the next line. This overflow is indicated by a chip with the modifier `.pf-m-overflow` that acts as a button to show/hide the overflown chips by expanding the height of the container they belong to.
-
-The chip group requires the [chip component](/documentation/core/components/chip).
-
-**All single chip accessibility and usage requirements apply.**
-
-## Accessibility
-
-| Attributes for closable chip group button | Applied to | Outcome |
-| -- | -- | -- |
-| `aria-label="[button label text]"` | `.pf-c-chip-group__close > button` |  Provides an accessible name for a chip group close when an icon is used instead of text. Required when an icon is used with no supporting text. **Required** |
-| `aria-labelledby="[id value of .pf-c-chip-group__close > button] [id value of .pf-c-chip-group__label]"` | `.pf-c-chip-group__close > button` | Provides an accessible name for the button. **Required** |
-
-### Usage
-| Class | Applied to | Outcome |
-| -- | -- | -- |
-| `.pf-c-chip-group__close` | `<div>` | Initiates the container used to close chip group. **Required.** |
-
-The chip group requires the [chip component](/documentation/core/components/chip).
-
-```hbs title=Multi-select-collapsed
-{{#> chip-group chip-group--id="select-collapsed"}}
+```hbs title=Chip-group-with-categories-overflow
+{{#> chip-group chip-group--modifier="pf-m-category" chip-group--id="toolbar-collapsed"}}
```

Comment:
`toolbar-collapsed` is used in another example.

Reply:
updated the ID of all of them to reflect what is happening